tools/merlin/merlin.xml

```diff
@@ -0,0 +1,471 @@
+		    <has_text text= " 1 (F)               2 (F)              5 (2,1)             4 (2,1)" />
+		    <has_text text= " A : B               C : D               C : A               D : B" />
+		    <has_text text= "A : B               C : D               C | A               D | B" />
+		    <has_text text= "A : B               C : D               C | A               D | B" />
+		</assert_contents>
+	    </output>
+	    <output name="errors">
+		<assert_contents>
+		    <has_text text= "118          3  rs9510743     0.0239"/>
+		    <has_text text= "197          1  rs9580624     0.0230"/>
+		    <has_text text= "296          1  rs9510743    0.00914" />
+		</assert_contents>
+	    </output>
+	</test>
+    </tests>
+    <help>
+	<![CDATA[
+		 MERLIN 1.1.2 - (c) 2000-2007 Goncalo Abecasis
+
+References for this version of Merlin:
+
+Abecasis et al (2002) Nat Gen 30:97-101        [original citation]
+Fingerlin et al (2004) AJHG 74:432-43          [case selection for association studies]
+Abecasis and Wigginton (2005) AJHG 77:754-67   [ld modeling, parametric analyses]
+Fingerlin et al (2006) Gen Epidemiol 30:384-96 [sex-specific maps]
+Chen and Abecasis (2007) AJHG 81:913-26        [qtl association analysis, qtl simulation]
+
+
+The following parameters are in effect:
+Data File :      merlin.dat (-dname)
+Pedigree File :      merlin.ped (-pname)
+Missing Value Code :         -99.999 (-xname)
+Map File :      merlin.map (-mname)
+Allele Frequencies : ALL INDIVIDUALS (-f[a|e|f|m|file])
+Random Seed :          123456 (-r9999)
+
+Data Analysis Options
+General : --error, --information, --likelihood, --model [param.tbl]
+IBD States : --ibd, --kinship, --matrices, --extended, --select
+NPL Linkage : --npl, --pairs, --qtl, --deviates, --exp
+VC Linkage : --vc, --useCovariates, --ascertainment, --unlinked [0.00]
+Association : --infer, --assoc, --fastAssoc, --filter, --custom [cov.tbl]
+Haplotyping : --best, --sample, --all, --founders, --horizontal
+Recombination : --zero, --one, --two, --three, --singlepoint
+Positions : --steps, --maxStep, --minStep, --grid, --start, --stop
+LD Clusters : --clusters [], --distance, --rsq, --cfreq
+Limits : --bits [24], --megabytes, --minutes
+Performance : --trim, --noCoupleBits, --swap, --smallSwap
+Output : --quiet, --markerNames, --frequencies, --perFamily, --pdf,
+--tabulate, --prefix [merlin]
+Simulation : --simulate, --reruns, --save, --trait []
+
+
+FATAL ERROR - The datafile merlin.dat cannot be opened
+
+Common causes for this problem are:
+
+1. You might not have used the correct options to specify input file names, please check the program documentation for information on how to do this
+
+2. The file doesn't exist or the filename might have been misspelt.
+
+3. The file exists but it is being used by another program which you will need to close before continuing
+
+4. The file is larger than 2 GB and you haven't compiled this application with large file support.
```

Comment:
The help text needs to explain what the tool does to users without bombarding them with technical information.

Could you put together a small paragraph on what Merlin is, based on the intro text [here](http://csg.sph.umich.edu/abecasis/merlin/index.html) and the tool abstract [here](https://www.ncbi.nlm.nih.gov/pubmed/11731797)